scipy/cluster/hierarchy.py

```diff
@@ -3491,17 +3490,16 @@ def _dendrogram_calculate_info(Z, p, truncate_mode,
     Returns
     -------
     A tuple (left, w, h, md), where:
+        * left is the independent variable coordinate of the center of the
+          the U of the subtree
 
-      * left is the independent variable coordinate of the center of the
-        the U of the subtree
-
-      * w is the amount of space used for the subtree (in independent
-        variable units)
+        * w is the amount of space used for the subtree (in independent
+          variable units)
 
-      * h is the height of the subtree in dependent variable units
+        * h is the height of the subtree in dependent variable units
 
-      * md is the ``max(Z[*,2]``) for all nodes ``*`` below and including
-        the target node.
+        * md is the ``max(Z[*,2]``) for all nodes ``*`` below and including
+          the target node.
```

Comment:
Is this the right/standard way to create bullet lists in `numpydoc`? I always thought it was:

```
Description of something

* first point
* second point
* third point

```

What I see here is:

```
Description of something
  * first point

  * second point

  * third point

```

This probably doesn't matter much here as it isn't rendered on the website.

Reply:
Probably not, right now I'm only making sure that Numpydoc does not crash when parsing it, and I basically do : 

 Parse -> Fix -> reformat -> ReParse , and  make sure the -re-parsing give the same data sctructure than after the "fix" step.

Numpydoc "sees" the following for the return section regardless of the whitespaces.

```
In [6]: ds.NumpyDocString(s)._parsed_data
Out[6]:
{'Signature': '',
...
 'Parameters': [],
 'Returns': [Parameter(name='', type='A tuple (left, w, h, md), where:', desc=['* left is the independent variable coordinate of the center of the', '  the U of the subtree', '', '* w is the amount of space used for the subtree (in independent', '  variable units)', '', '* h is the height of the subtree in dependent variable units', '', '* md is the ``max(Z[*,2]``) for all nodes ``*`` below and including', '  the target node.'])],
 'Yields': [],
 ...
```

Reply:
> Parse -> Fix -> reformat -> ReParse , and make sure the -re-parsing give the same data sctructure than after the "fix" step.

Oh, OK. That makes sense.

> Numpydoc "sees" the following for the return section regardless of the whitespaces.

Hmm, didn't know `numpydoc` doesn't care about whitespaces. Thanks for the clarification!